pkg/resource/backup/hooks.go

```diff
@@ -0,0 +1,60 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package backup
+
+import (
+	"errors"
+
+	ackrequeue "github.com/aws-controllers-k8s/runtime/pkg/requeue"
+
+	"github.com/aws-controllers-k8s/dynamodb-controller/apis/v1alpha1"
+)
+
+var (
+	// TerminalStatuses are the status strings that are terminal states for a
+	// backup.
+	TerminalStatuses = []v1alpha1.BackupStatus_SDK{}
```

Comment:
this is meant to be empty?

Reply:
There are no terminal statuses for `Backup`, for now.. https://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BackupDetails.html

Reply:
k.